### PR TITLE
Review payment & payout request schemas

### PR DESCRIPTION
**Pull request by martinseco**

### CODE REVIEW

**Comment by armandojaleo** [comment]
Probably some test are failing by the API, must we skip them?

File: common/common.go [hunk 1]
```diff
@@ -155,12 +155,14 @@ type (
 		Type           AccountHolderIdentificationType `json:"type,omitempty"`
 		Number         string                          `json:"number,omitempty"`
 		IssuingCountry Country                         `json:"issuing_country,omitempty"`
-		DateOfExpiry   AccountHolderType               `json:"date_of_expiry,omitempty"`
```
Comment (Armando-Ibarra) [inline common/common.go:158]: This is unfortunate

File: payments/abc/payments.go [hunk 2]
```diff
@@ -11,51 +11,51 @@ import (
...
-		PreviousPaymentId string                      `json:"previous_payment_id,omitempty"`
-		Risk              *payments.RiskRequest       `json:"risk,omitempty"`
-		SuccessUrl        string                      `json:"success_url,omitempty"`
-		FailureUrl        string                      `json:"failure_url,omitempty"`
-		PaymentIp         string                      `json:"payment_ip,omitempty"`
-		ThreeDsRequest    *payments.ThreeDsRequest    `json:"3ds,omitempty"`
-		PaymentRecipient  *payments.PaymentRecipient  `json:"recipient,omitempty"`
-		Metadata          map[string]interface{}      `json:"metadata,omitempty"`
-		Processing        map[string]interface{}      `json:"processing,omitempty"`
+		Source            payments.PaymentSource       `json:"source,omitempty"`
```
Comment (Armando-Ibarra) [inline payments/abc/payments.go:14]: Remind me why this is not a pointer?

File: payments/payments.go [hunk 3]
```diff
@@ -241,44 +241,50 @@ type (
...
-		AccountNumber string         `json:"account_number,omitempty"`
-		CountryCode   common.Country `json:"country_code,omitempty"`
-		Zip           string         `json:"zip,omitempty"`
-		FirstName     string         `json:"first_name,omitempty"`
-		LastName      string         `json:"last_name,omitempty"`
+		DateOfBirth   string          `json:"dob,omitempty"`
+		AccountNumber string          `json:"account_number,omitempty"`
+		Address       *common.Address `json:"address,omitempty"`
+		CountryCode   common.Country  `json:"country_code,omitempty"`
+		// Deprecated: should use common.Address.Zip instead
```
Comment (Armando-Ibarra) [inline payments/payments.go:248]: Same comment as Ruby, this is still available for previous

**Review by Armando-Ibarra — APPROVED** [review]

**Review by Armando-Ibarra — APPROVED** [review]

**Pull request merged**